### PR TITLE
Fix for issue #746 - removed invalid options for filesystem (STANDARD…

### PR DESCRIPTION
**Pull request by mattstreet-nag**
… and PREMIUM are now deprecated) and added ENTERPRISE.

Added files to .gitignore to ensure safer commits.

### Submission Checklist

* [ ] Have you installed and run this change against pre-commit? (`pre-commit
  install`)
* [ ] Are all tests passing? (`make tests`)
* [ ] Have you written unit tests to cover this change?
* [ ] Is unit test coverage still above 80%?
* [ ] Have you updated all applicable documentation?
* [ ] Have you followed the guidelines in our Contributing document?

### CODE REVIEW

File: community/front-end/ofe/website/ghpcfe/models.py [hunk 1]
```diff
@@ -1230,11 +1230,10 @@ class GCPFilestoreFilesystem(Filesystem):
 
     FILESTORE_TIER = (
         ("u", "TIER_UNSPECIFIED"),
```
Comment (heyealex) [inline community/front-end/ofe/website/ghpcfe/models.py:1232]: What is the behavior when the tier is unspecified?

Comment (mattstreet-nag) [reply community/front-end/ofe/website/ghpcfe/models.py:1232]: @heyealex 

Terraform creation fails, as expected with the same validator in #746, and filesystem is not created.
<pre>
Error: Invalid value for variable

  on main.tf line 27, in module "fs1":
  27:   filestore_tier       = "TIER_UNSPECIFIED"
    ├────────────────
    │ var.filestore_tier is "TIER_UNSPECIFIED"
</pre>

Would agree this is a bit pointless in the Front End - was probably added for completeness.

Comment (heyealex) [reply community/front-end/ofe/website/ghpcfe/models.py:1232]: Could it be removed as well without breaking anything? I can see the need for a null value, but a user shouldn't be able to select it.

Comment (heyealex) [reply community/front-end/ofe/website/ghpcfe/models.py]: I just removed it and ran a quick test, everything worked as expected.

**Review by heyealex — APPROVED** [review]

**Pull request merged**